milner_desvars/diff-drive.md

```diff
@@ -0,0 +1,5 @@
+# how a differential-drive robot works and how to control it
```

Comment:
Please add more information about the unicycle model and how that relates to the [Twist](https://docs.ros.org/api/geometry_msgs/html/msg/Twist.html) message.

Reply:
That information can be related to what you already wrote.